RecoTracker/MkFit/plugins/createPhase1TrackerGeometry.cc

```diff
@@ -0,0 +1,372 @@
+//-------------------
+// Phase1 tracker geometry
+//-------------------
+
+#include "Config.h"
+#include "Debug.h"
+#include "TrackerInfo.h"
+#include "mkFit/IterationConfig.h"
+#include "mkFit/HitStructures.h"
+
+#include "HitSelectionWindowsPhase1.h"
+
+#include <functional>
+
+using namespace mkfit;
+
+namespace {
+#include "createPhase1TrackerGeometryAutoGen.acc"
+
+  void setupSteeringParamsIter0(IterationConfig &ic) {
+    ic.m_region_order[0] = TrackerInfo::Reg_Transition_Pos;
+    ic.m_region_order[1] = TrackerInfo::Reg_Transition_Neg;
+    ic.m_region_order[2] = TrackerInfo::Reg_Endcap_Pos;
+    ic.m_region_order[3] = TrackerInfo::Reg_Endcap_Neg;
+    ic.m_region_order[4] = TrackerInfo::Reg_Barrel;
+
+    {
+      SteeringParams &sp = ic.m_steering_params[TrackerInfo::Reg_Endcap_Neg];
+      sp.reserve_plan(3 + 3 + 6 + 18);
```

Comment:
FPIX : 3+3
TID : 3+3 = 6
TEC : 9+9=18 
?

Reply:
but, I'm confused (sorry)
on the negative side there are only 3+3+9 ....can you please help me in understanding ? thanks

Reply:
Comments added to explain each number.